Commands/Commands/queue.js

```diff
@@ -607,16 +607,22 @@ function switchIDs (og, bot) {
 function deleteFromUV (UVID, uvClient, bot) {
   uvClient.v1.loginAsOwner().then(i => {
     i.delete(`forums/${config.uservoice.forumId}/suggestions/${UVID}.json`).catch((e) => {
+      if (e.statusCode === 404) {
+        bot.Channels.find(c => c.name === 'admin-queue').sendMessage('That suggestion doesn\'t seems to exist, (someone must have beat you too it).').then(o => {
+              setTimeout(() => bot.Messages.deleteMessages([o.id, msg.id], bot.Channels.find(c => c.name === 'admin-queue').id), config.timeouts.messageDelete)
```

Comment:
- Supply the actual message objects, so you can leave the channel ID out.
- Format please.